packages/core/src/fragments/IfcGeometryTiler/src/base-types.ts

```diff
@@ -1,30 +1,39 @@
-/**
- * A dictionary of geometries streamed from a server. Each geometry is identified by a unique number (id), and contains information about its bounding box, whether it has holes, and an optional file path for the geometry data.
- */
+interface GeometrySettings {
+  /** The bounding box of the geometry as a Float32Array. */
+  boundingBox: Float32Array;
+  /** A boolean indicating whether the geometry has holes. */
+  hasHoles: boolean;
+}
+
 export interface StreamedGeometries {
-  [id: number]: {
-    /** The bounding box of the geometry as a Float32Array. */
-    boundingBox: Float32Array;
-    /** A boolean indicating whether the geometry has holes. */
-    hasHoles: boolean;
-    /** An optional file path for the geometry data. */
-    geometryFile?: string;
-  };
+  [id: number]: GeometrySettings;
 }
 
 /**
  * A streamed asset, which consists of multiple geometries. Each geometry in the asset is identified by a unique number (geometryID), and contains information about its transformation and color.
  */
 export interface StreamedAsset {
-  /** The unique identifier of the asset. */
   id: number;
   /** An array of geometries associated with the asset. */
-  geometries: {
```

Comment:
using the generic `Array<>` instead of the `[]` prefix helps readability for complex array types. its easier to see that the field is an array. the `[]` are easily overlooked in these cases.